docs/contributing/contributing-to-dataline/updating-documentation.md

```diff
@@ -30,7 +30,7 @@ To update with GitBook, follow these instructions:
 
 1. Create a [new variant](https://docs.gitbook.com/editing-content/variants#create-a-variant). This will create a new branch on [GitHub](https://github.com/datalineio/dataline) with the same name
 2. Modify the documentation in that new variant
-3. Save & merge regularly
+3. Save & merge regularly. Each time you merge, your changes will propagated to you branch. It can take a couple minutes for the changes to show up in [GitHub](https://github.com/datalineio/dataline).
```

Comment:
```suggestion
3. Save & merge regularly. Each time you merge, your changes will propagated to your branch. It can take a couple minutes for the changes to show up in [GitHub](https://github.com/datalineio/dataline). Alternatively, you can push changes directly to the `.md` file via git, and those changes will be reflected immediately in git. 
```

Reply:
Fair enough -- we can mention that somewhere else if needed